rotkehlchen/rotkehlchen.py

```diff
@@ -356,6 +356,7 @@ def unlock_user(
             evm_tx_decoder=self.evm_tx_decoder,
             deactivate_premium=self.deactivate_premium_status,
             query_balances=self.query_balances,
+            activate_premium=self.activate_premium_status,
```

Comment:
maybe just for similarity you can put this two near each other? just something really minor (also update the position where this function is called)